app.py

```diff
@@ -251,7 +251,15 @@ def handle_error(e):
     return jsonify(error=str(e)), 500
 
 
-if __name__ == '__main__':
+def start_debug_app():
     sslify = SSLify(app)
```

Comment:
Do we still have SSL serving in container?

Reply:
No, Docker is not actually using `app.py` as `main`, it uses `server.py`. This whole method might be not needed anymore.
For me, this isn't required anymore, but not sure if @interro isn't using this in his local machine?